docs/src/content/changelogs/0.3.md

```diff
@@ -0,0 +1,43 @@
+---
+version: '0.3'
+date: 'Dec 1 22'
+pr: 10
+---
+
+## API Changes
+
+### Oprish API Changes
+
+- Due to rate limits being a per instance configuration thing, a new [`/rate_limits`
+  route](/404) has been added besides the already pre-existing
```

Comment:
I'm a bit confused as to what these hyperlinks are for? Are they missing a link to go to?

Reply:
They've been removed as of 0.3.3 (this version)

Reply:
The ones below seem to be an attempt at links
```md
The [`get_instance_info`] route now has a rate-limit.
```

Reply:
Oh, docs have a custom plugin that follows what `docs.rs` does- essentially autolinking stuff in the "[`foo`]" format. Think of it like ":class:`bar`" for sphinx but it only covers autodocumented items

Reply:
~~oh, cringe~~ /s